adapters/conversant/conversant.go

```diff
@@ -18,6 +19,11 @@ type ConversantAdapter struct {
 }
 
 func (c *ConversantAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	cnvrRequest := *request
+	//Backend needs USD or it will reject the request
+	if cnvrRequest.Cur != nil && len(cnvrRequest.Cur) > 0 && cnvrRequest.Cur[0] != "USD" {
+		cnvrRequest.Cur = []string{"USD"}
+	}
```

Comment:
Please add the json test cases for this change.

Reply:
I have added this to one of the test cases